frontend/coprs_frontend/coprs/views/apiv3_ns/schema.py

```diff
@@ -0,0 +1,560 @@
+}
+
+pagination_model = api.model("Pagination", pagination_schema)
+
+project_chroot_schema = {
+    "mock_chroot": mock_chroot_field,
+    "ownername": ownername_field,
+    "projectname": projectname_field,
+    "comps_name": comps_name_field,
+    "additional_repos": additional_repos_field,
+    "additional_packages": additional_packages_field,
+    "additional_modules": additional_modules_field,
+    "with_opts": with_opts_field,
+    "without_opts": without_opts_field,
+    "delete_after_days": delete_after_days_field,
+    "isolation": isolation_field,
+}
+
+project_chroot_model = api.model("ProjectChroot", project_chroot_schema)
+
+repo_schema = {
+    "baseurl": String,
+    "id": String(example="copr_base"),
+    "name": String(example="Copr repository"),
+}
+
+repo_model = api.model("Repo", repo_schema)
+
+project_chroot_build_config_schema = {
+    "chroot": mock_chroot_field,
+    "repos": List(Nested(repo_model)),
+    "additional_repos": additional_repos_field,
+    "additional_packages": additional_packages_field,
+    "additional_modules": additional_modules_field,
+    "enable_net": enable_net_field,
+    "with_opts":  with_opts_field,
+    "without_opts": without_opts_field,
+    "isolation": isolation_field,
+}
+
+project_chroot_build_config_model = \
+    api.model("ProjectChrootBuildConfig", project_chroot_build_config_schema)
+
+source_dict_scm_schema = {
+    "clone_url": clone_url_field,
+    "committish": committish_field,
+    "source_build_method": source_build_method_field,
+    "spec": spec_field,
+    "subdirectory": subdirectory_field,
+    "type": scm_type_field,
+}
+
+source_dict_scm_model = api.model("SourceDictSCM", source_dict_scm_schema)
+
+source_dict_pypi_schema = {
+    "pypi_package_name": pypi_package_name_field,
+    "pypi_package_version": pypi_package_version_field,
+    "spec_generator": pypi_spec_generator_field,
+    "spec_template": pypi_spec_template_field,
+    "python_versions": pypi_versions_field,
+}
+
+source_dict_pypi_model = api.model("SourceDictPyPI", source_dict_pypi_schema)
+
+source_package_schema = {
+    "name": packagename_field,
+    "url": source_package_url_field,
+    "version": source_package_version_field,
+}
+
+source_package_model = api.model("SourcePackage", source_package_schema)
+
+build_schema = {
+    "chroots": chroots_field,
+    "ended_on": ended_on_field,
+    "id": id_field,
+    "is_background": is_background_field,
+    "ownername": ownername_field,
+    "project_dirname": project_dirname_field,
+    "projectname": projectname_field,
+    "repo_url": repo_url_field,
+    "source_package": Nested(source_package_model),
+    "started_on": started_on_field,
+    "state": state_field,
+    "submitted_on": submitted_on_field,
+    "submitter": submitter_field,
+}
+
+build_model = api.model("Build", build_schema)
+
+package_builds_schema = {
+    "latest": Nested(build_model, allow_null=True),
+    "latest_succeeded": Nested(build_model, allow_null=True),
```

Comment:
We miss ["builds" field](https://github.com/fedora-copr/copr/blob/a0bee3f1fae88c638c9d27082572d3d7e14fce82/frontend/coprs_frontend/coprs/models.py#L928-L929) here.

Reply:
I don't think the `with_all_builds` option is exposed to the API

```
Your branch is up to date with 'origin/main'
$ git grep with_all_builds frontend/
frontend/coprs_frontend/coprs/models.py:    def to_dict(self, with_latest_build=False, with_latest_succeeded_build=False, with_all_builds=False):
frontend/coprs_frontend/coprs/models.py:        if with_all_builds:
```

Reply:
Indeed
```
copr --debug get-package --name python-backoff --with-all-builds @copr/copr-dev 
#  Debug log enabled  #
[10:48:13] {/usr/lib/python3.11/site-packages/urllib3/connectionpool.py:228} DEBUG - Starting new HTTP connection (1): localhost:55555
[10:48:13] {/usr/lib/python3.11/site-packages/urllib3/connectionpool.py:456} DEBUG - http://localhost:55555 "GET /api_3/package?ownername=%40copr&projectname=copr-dev&packagename=python-backoff&with_latest_build=False&with_latest_succeeded_build=False HTTP/1.1" 200 458
[10:48:13] {/usr/lib/python3.11/site-packages/urllib3/connectionpool.py:228} DEBUG - Starting new HTTP connection (1): localhost:55555
[10:48:13] {/usr/lib/python3.11/site-packages/urllib3/connectionpool.py:456} DEBUG - http://localhost:55555 "GET /api_3/build/list?ownername=%40copr&projectname=copr-dev&packagename=python-backoff HTTP/1.1" 200 809
```